CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* Fri Mar 22 2019 Joseph Sharkey <shark.bruhaha@gmail.com> - 3.15.0-0
+- Marked bracketize puppet 3 function as deprecated.
```

Comment:
Need to note:  "This function will be removed in a future release, unless SIMP receives a request from users to convert it to a namespaced Puppet 4.x function'.
This matches other function deprecation CHANGELOG entries.

Reply:
It is converted to a puppet 4 function...

Reply:
*DOH!* I forgot to add the files...

Fixing in push soon

Reply:
Function and its test were not added to git, so not visible in the PR